components/monitoring/logging/base/configure-logging/configure-logforwarder.yaml

```diff
@@ -14,10 +14,33 @@ spec:
       secret:
         name: cloudwatch-forwarder
       type: cloudwatch
+    - name: splunk-receiver-application
+      secret:
+        name: log-forwarder-splunk-rhtap-application-secret
+      type: splunk
+      url: https://http-inputs-rhcorporate.splunkcloud.com
+    - name: splunk-receiver-audit
+      secret:
+        name: log-forwarder-splunk-rhtap-audit-secret
+      type: splunk
+      url: https://http-inputs-rhcorporate.splunkcloud.com
```

Comment:
putting configuration in base - while cleaner, will also apply it for DEV clusters - not suer we want that...

Reply:
@ifireball We haven't added any configuration for dev so far so lets take care of the development environment at a later stage.  I'm not even sure if I can assume DEV clusters will have access to the Vault or HEC endpoints.

In the scope of this PR, the desired changes apply to staging and production.

Reply:
I'm not saying we should apply the configuration for DEV, I'm saying we need to make sure we're not applying it without intending to

Reply:
I don't think we do as long as there is no `development` path.
There was also CloudWatch configuration before without any implicit development configuration.

Reply:
@ifireball I've created a `development` folder and added a `kustomization.yaml` file with no resources from base.
That way nothing will be created in the development environment.